### ISSUE TITLE
TLS

### ISSUE THREAD

**Issue by zelgard** [meganz/MEGAcmd#700]
Can self signed certificates be used for setting up WebDaV?

**Comment by polmr** [comment]
Yes: https://github.com/meganz/MEGAcmd/blob/master/contrib/docs/WEBDAV.md#https

However, bear in mind that the client application needs to be able to cope with those
